cmd/kube-apiserver/app/server.go

```diff
@@ -655,7 +652,9 @@ func BuildStorageFactory(s *options.ServerRunOptions, apiResourceConfig *servers
 	return storageFactory, nil
 }
 
-func defaultOptions(s *options.ServerRunOptions) error {
+// DefaultOptions set default ServerRunOptions.
+// Should be called after kube-apiserver flags parsed.
+func DefaultOptions(s *options.ServerRunOptions) error {
```

Comment:
this is a "complete func" as we have it for the config. Is it worth to wrap it into a private `defaultedServerRunOptions`?

Reply:
Done in last commit